src/Jobs/Update/CleanupProductTargetCountriesJob.php

```diff
@@ -0,0 +1,55 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Jobs\Update;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Jobs\AbstractProductSyncerBatchedJob;
+use Automattic\WooCommerce\GoogleListingsAndAds\Product\ProductSyncerException;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class CleanupProductTargetCountriesJob
+ *
+ * Deletes the previous list of target countries which was in use before the
+ * Global Offers option became available.
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Jobs\Update
+ */
+class CleanupProductTargetCountriesJob extends AbstractProductSyncerBatchedJob {
```

Comment:
I think it's now time to add the `@since` annotation to all our newly added classes and functions. Same for the `PluginUpdate` job and the other methods added in this PR.